getting-started/services.md

```diff
@@ -7,13 +7,17 @@ content_group: Getting started
 ---
 
 import Install from "versionContent/_partials/_cloud-installation.mdx";
-import CreateService from "versionContent/_partials/_cloud-create-service.mdx";
 import Connect from "versionContent/_partials/_cloud-connect.mdx";
-import CloudIntro from "versionContent/_partials/_cloud-intro.mdx";
+import CreateAHypertable from "versionContent/_partials/_create-hypertable.mdx";
 
 # Create your first Timescale service
 
-You use Timescale Console to create and manage Timescale services. Each service is a single 100% PostgreSQL database with usage-based storage.
+You use Timescale Console to create and manage `Ops` for your Timescale Cloud services, and securely manage 
```

Comment:
This reads kinda funky to me, the `manage Ops` part.

What do you think of introducing it like:
- There are two modes in the Timescale Console, depending on what you're trying to accomplish
- Ops mode (short for operations) is where you can manage your services, add replicas, configure compression, change parameters, etc. 
- Data mode is where you can interact with your data, powered by PopSQL: write queries with autocomplete, visualize data with charts and dashboards, schedule queries and dashboards to create alerts or recurring reports, share queries and dashboards, and more.

The screenshots could also be side-by-side, and under each screenshot, you could describe each mode. That would help people visualize "ok Ops mode is for this kinda stuff, Data mode is for that kinda stuff". I used [v0.dev](https://v0.dev) to generate a sample mockup to convey what I mean:
<img width="1358" alt="image" src="https://github.com/user-attachments/assets/71f78c38-5e9d-4fee-ba01-73031882ba30">